### PR TITLE
fixed some broken things

### PR DESCRIPTION
**Pull request by a1exsDev**
Fixed broken commands, reddit, chan, and replaced some deprecated web functions with more modern ones

### CODE REVIEW

File: terminal/term.js [hunk 1]
```diff
@@ -85,7 +85,7 @@ function loadConfig() {
     if (files[".config"]) {
         userConfig = formatToJSON(files[".config"]);
 
-        //then load any similarities into terminal and proceed as normal
+        //then load any simifilarities into terminal and proceed as normal
```
Comment (garzaa) [inline terminal/term.js:88]: typo

File: terminal/term.js [hunk 2]
```diff
@@ -123,8 +123,10 @@ function addListeners() {
             handleInput($("#input").html());
             inputIndex = 0;
         } else if (key === 38) { //up arrow
-            document.getElementById("input").innerHTML = lastInputs[inputIndex];
-            inputIndex < lastInputs.length-1 ? inputIndex++ : true;
+            if (lastInputs[inputIndex] != undefined) {
```
Comment (garzaa) [inline terminal/term.js:126]: could you just check with a simple boolean `if (lastInputs[inputIndex])` instead? or are you specifically checking for undefined here

File: terminal/term.js [hunk 3]
```diff
@@ -626,6 +578,20 @@ function getBrowser() {
...
+    var nAgt = navigator.userAgent;
+    var browserEngine = 'Unknown';
+    if (nAgt.includes("Chrome")) {
+        browserEngine = "Blink";
+    } else if (nAgt.includes("Firefox")) {
+        browserEngine = "Gecko"
+    } else if (nAgt.includes("Safari")) {
+        browserEngine = "Webkit"
+    } else if (nAgt.includes("Trident")) {
+        browserEngine = "Trident"
```
Comment (garzaa) [inline terminal/term.js:591]: cool, i hadn't heard of this before

File: terminal/term.js [hunk 4]
```diff
@@ -171,7 +174,7 @@ function handleInput(rawInput) {
 
     //do everything else here
     if (preCheck(rawInput)) {
-        return;
+         return;
```
Comment (garzaa) [inline terminal/term.js:177]: nice

**Review by garzaa — APPROVED** [review]
lgtm, a couple nitpicks you should look at before merging

File: terminal/term.js (hunk 2 above)
Comment (a1exsDev) [reply terminal/term.js:126]: It needs to check for undefined because if not, undefined will pop up if there is nothing in the command history.

Comment (garzaa) [reply terminal/term.js:126]: I see, that makes sense.

**Comment by a1exsDev** [comment]
fixed them, you can push them to master now

**Pull request merged**